src/DssSpell.sol

```diff
@@ -41,35 +55,172 @@ contract DssSpellAction is DssAction {
+        DssExecLib.setIlkAuctionDuration("ETH-B", 4 hours);
+        DssExecLib.setIlkBidDuration("BAT-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("BAT-A", 4 hours);
+        DssExecLib.setIlkBidDuration("WBTC-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("WBTC-A", 4 hours);
+        DssExecLib.setIlkBidDuration("KNC-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("KNC-A", 4 hours);
+        DssExecLib.setIlkBidDuration("ZRX-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("ZRX-A", 4 hours);
+        DssExecLib.setIlkBidDuration("MANA-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("MANA-A", 4 hours);
+        DssExecLib.setIlkBidDuration("USDT-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("USDT-A", 4 hours);
+        DssExecLib.setIlkBidDuration("COMP-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("COMP-A", 4 hours);
+        DssExecLib.setIlkBidDuration("LRC-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("LRC-A", 4 hours);
+        DssExecLib.setIlkBidDuration("LINK-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("LINK-A", 4 hours);
+        DssExecLib.setIlkBidDuration("BAL-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("BAL-A", 4 hours);
+        DssExecLib.setIlkBidDuration("YFI-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("YFI-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNI-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNI-A", 4 hours);
+        DssExecLib.setIlkBidDuration("RENBTC-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("RENBTC-A", 4 hours);
+        DssExecLib.setIlkBidDuration("AAVE-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("AAVE-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNIV2DAIETH-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNIV2DAIETH-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNIV2WBTCETH-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNIV2WBTCETH-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNIV2USDCETH-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNIV2USDCETH-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNIV2ETHUSDT-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNIV2ETHUSDT-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNIV2LINKETH-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNIV2LINKETH-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNIV2UNIETH-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNIV2UNIETH-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNIV2WBTCDAI-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNIV2WBTCDAI-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNIV2AAVEETH-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNIV2AAVEETH-A", 4 hours);
+        DssExecLib.setIlkBidDuration("UNIV2DAIUSDT-A", 4 hours);
+        DssExecLib.setIlkAuctionDuration("UNIV2DAIUSDT-A", 4 hours);
```

Comment:
We should do something like:
```
        // Loop over all ilks
        //
        bytes32[] memory ilks = DssExecLib.reg().list();

        for (uint i = 0; i < ilks.length; i++) {
            // Decrease the bid duration (ttl) and max auction duration (tau) from 6 to 4 hours to all the ilks with liquidation on
            if (DssExecLib.flip(ilks[i]).wards(DssExecLib.flipperMom()) == 1) {
                DssExecLib.setIlkBidDuration(ilks[i], 4 hours);
                DssExecLib.setIlkAuctionDuration(ilks[i], 4 hours);
            }
        }
```

Reply:
As we discussed in the chat, I had thought to implement this exact same thing. But we would have 33 extra external calls and 64 extra SLOADs.
If the majority prefers this way I'm not strongly against it but it is quite more inefficient.

Reply:
Why aren't we changing the non-liquidatable ones? Might be nice to have all of the values in sync, even if they aren't being used.

Reply:
I agree, but that's what the vote said. In fact we still have some unliquidatable ones with the 3 days `tau`.

Reply:
Anyways, I'd probably have written it as a loop but I'm fine with this as-is. Verified values against test collaterals.